frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -88,6 +92,11 @@ const Metric = (props: MetricProps) => {
         const mode = (props.type === "none") ? [] : ["gauge"];
         showValue && mode.push("number");
         (delta !== undefined) && mode.push("delta");
+        const deltaIncreasing = deltaColor ? {
+            color: deltaColor == "invert" ? "#FF4136" : deltaColor } : undefined
```

Comment:
I found the default color in their [documentation](https://plotly.com/javascript/reference/indicator/). 

I checked the library and it's hardcoded. Is there any way to get these values?
https://github.com/plotly/plotly.js/blob/b6411b58357ecdc1c1518dede25060bac7d6cd41/src/constants/delta.js#L4

I solved the test problem. I just needed to make the delta value negative instead of value.